src/adapter/services/LegacyObjectAPIInterceptor.js

```diff
@@ -52,7 +52,7 @@ define([
 
             oldStyleObject.getCapability('mutation').mutate(function () {
                 return utils.toOldFormat(newStyleObject);
-            });
+            }, newStyleObject.modified);
```

Comment:
Ah cool! I had made a change for this in MutationCapablity.js but this is probably a better spot.